civihr_default_theme/scss/base/_custom.scss

```diff
@@ -3,16 +3,17 @@ legend {
   border-bottom: 1px solid $legend-border-color;
   color: $legend-color;
   display: block;
-  font-size: ($font-size-base * 1.5);
+  font-size: $font-size-base;
```

Comment:
we don't need to do this size manipulation anymore?

Reply:
Yes, as per the plan, we will not be using any custom font sizes anymore. Instead we have fetched all the different font sizes from the mockup, and only those are used.
If Jon suggests a different font size for a specific element, we will update the variables accordingly and use the same.